.azure-pipelines/native-unit-tests.yml

```diff
@@ -0,0 +1,45 @@
+variables:
+  buildConfiguration: debug
+
+jobs:
+
+- job: 'windows'
```

Comment:
Is the intent to add the Linux unit tests in this file as well?

Reply:
That was the plan, then I realized we don't run the C++ unit tests on Linux. Do you think it's a good idea to have Windows/Linux in the same file?

Reply:
Heh, I didn't realize we didn't run the C++ unit tests on Linux. I think having them in the same file could make sense, but I don't have a strong preference either way.